ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/components/gridConfig/gridConfig.html

```diff
@@ -9,15 +9,18 @@
         <img itemprop="thumbnail" width="192" src="${model.thumbnail}" alt="">
     </td>
 
-    <td class="foundation-collection-item-title" is="coral-table-cell">
+    <td class="foundation-collection-item-title collection-item_link_type" is="coral-table-cell">
```

Comment:
Is `collection-item_link_type` the class name that you created with your own hand? If so, you might want to read https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/ about the best practices of CSS naming